### PR TITLE
Add open graph image

### LINKED ISSUE #283: Set the feature image of a post as the OG image for searching
We currently have no OG (social data) in the header of the site. It would be good to add them.

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Set featured image to be displayed by Facebooks open graph meta tag

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Choose a post with a featured image (a thumbnail on the card) and if possible share on a facebook account. It should feature the same image
- Choose a post with no featured image and if possible share on a facebook account. It should feature the default insights logo on a suru background.

## Issue / Card

Fixes #283

### CODE REVIEW

**Comment by anthonydillon** [comment]
While we are here @steverydz could you go a step further and add:

```
<meta property="og:title" content="..." /> // Title of the page
<meta property="og:url" content="..." /> // Canonical URL incase people visit from newletters with tracking params
<meta property="og:description" content="..." /> // The excerpt
```

File: templates/layout.html [hunk 1]
```diff
@@ -5,6 +5,28 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% if post %}
```
Comment (anthonydillon) [inline templates/layout.html]: Does this need `if post && self`?

Comment (steverydz) [reply templates/layout.html]: I don't think so. The only block using `self` is title and is wrapped. Also isn't `self` implicit?

**Comment by steverydz** [comment]
@anthonydillon Done

File: templates/layout.html (hunk 1 above)
Comment (anthonydillon) [reply templates/layout.html]: Oh I see its outside of this block. Read it wrong.

**Pull request merged**